### PR TITLE
Rollback Azure Pipelines Jobs To Travis

### LINKED ISSUE #1981: [Feature] Azure Pipelines Follow-Up
Follow-up work for #1969 .

Currently only the linux Docker jobs have been migrated to azure pipelines. 

Still have osx and total docker jobs to migrate over from Travis CI.

### PR DESCRIPTION
**Pull request by bmhan12**
This PR:
- Moves Azure jobs back to Travis
- Deletes Azure configuration file
- Adds a `--reduce-install-logs` flag to do a `make install` instead of ` make install VERBOSE=1` to preventing hitting [log length error](https://docs.travis-ci.com/user/common-build-problems/#log-length-exceeded).
- Relates to #2029, Closes #1981

### CODE REVIEW

File: scripts/build_test_helper.sh [hunk 1]
```diff
@@ -78,7 +78,13 @@ if [[ "$*" == *--build-exe-only* ]]; then
...
 else
   or_die make -j $(nproc) VERBOSE=1
-  or_die make install VERBOSE=1
+
+  # Verbosity check for installation to prevent hitting Travis log limit
+  if [[ "$*" == *--reduce-install-logs* ]]; then
+    or_die make install
+  else
+    or_die make install VERBOSE=1
+  fi
```
Comment (TotoGaz) [inline scripts/build_test_helper.sh:87]: I like how those `if [[ "$*" == *...* ]];` reflect all the issues we happen to face during the project 😉

**Review by TotoGaz — APPROVED** [review]
I think that we could remove the `ci_build_and_test.sh` because it induces duplications and is now not really required.
We can do it in other PRs down the line.

**Comment by TotoGaz** [comment]
@bmhan12 Do you want I merge this right now?
I think it makes sense.

**Comment by bmhan12** [comment]
> @bmhan12 Do you want I merge this right now?

@TotoGaz Yes, that would be great.

**Pull request merged**